cardshowviewtakenpicturesview/src/main/java/br/com/stant/libraries/cardshowviewtakenpicturesview/camera/CameraPhotosAdapter.java

```diff
@@ -136,7 +153,21 @@ private boolean isNotLoading() {
         return mPhotos;
     }
 
-    class ItemViewHolder extends RecyclerView.ViewHolder {
+    public void setTouchHelper(ItemTouchHelper touchHelper) {
+        mTouchHelper = touchHelper;
+    }
+
+    @Override
+    public void onViewMoved(int oldPosition, int newPosition) {
+        if (newPosition < mPhotos.size() && oldPosition < mPhotos.size()) {
+            CameraPhoto targetCameraPhoto = mPhotos.get(oldPosition);
+            mPhotos.remove(oldPosition);
```

Comment:
add space between variables initialization and methods calls

Reply:
Fixed